tests/components/test_init.py

```diff
@@ -66,29 +69,6 @@ def test_toggle(self):
         self.hass.block_till_done()
         self.assertEqual(1, len(calls))
 
-    @patch('homeassistant.core.ServiceRegistry.call')
-    def test_turn_on_to_not_block_for_domains_without_service(self, mock_call):
```

Comment:
We should not remove this test. It means blocking is broken?

I think that we should change our services so that all of them are only finishing when the service is done. No longer use `create_task` - because although we do not care about the result, we do care about when the result is available.